visual-js/visual/src/utils.ts

```diff
@@ -1,4 +1,5 @@
-import { FullPageConfigIn } from './graphql/__generated__/graphql';
+import { type, Type } from 'arktype';
```

Comment:
We used `zod` in the past. What is the benefit of `arktype`?

Reply:
You tell me, it was originally your addition to `visual-wdio` I'm just moving it into the Visual package 😅

Reply:
forgot about that 🤦‍♂️